puppet/modules/kubernetes/manifests/dns.pp

```diff
@@ -31,18 +31,18 @@
   $post_1_10 = versioncmp($::kubernetes::version, '1.10.0') >= 0
 
   if $post_1_10 {
-    $app_name = 'core-dns'
```

Comment:
Is that name change down to sonobouy?

Reply:
Yes, guess we should consider it source of truth

Reply:
I am sure kube-dns is the correct name for it though